internal/net/client_grpc_dkg.go

```diff
@@ -8,78 +8,57 @@ import (
 	"github.com/drand/drand/protobuf/drand"
 )
 
-func (g *grpcClient) Propose(ctx context.Context, p Peer, in *drand.ProposalTerms, _ ...grpc.CallOption) (*drand.EmptyResponse, error) {
-	var resp *drand.EmptyResponse
+func (g *grpcClient) Command(ctx context.Context, p Peer, in *drand.DKGCommand, _ ...grpc.CallOption) (*drand.EmptyDKGResponse, error) {
+	var resp *drand.EmptyDKGResponse
 	c, err := g.conn(ctx, p)
 	if err != nil {
 		return nil, err
 	}
-	client := drand.NewDKGClient(c)
+	client := drand.NewDKGControlClient(c)
 	ctx, cancel := g.getTimeoutContext(ctx)
 	defer cancel()
-	resp, err = client.Propose(ctx, in)
+	resp, err = client.Command(ctx, in)
 	return resp, err
 }
 
-func (g *grpcClient) Abort(ctx context.Context, p Peer, in *drand.AbortDKG, _ ...grpc.CallOption) (*drand.EmptyResponse, error) {
-	var resp *drand.EmptyResponse
+func (g *grpcClient) Packet(ctx context.Context, p Peer, in *drand.GossipPacket, _ ...grpc.CallOption) (*drand.EmptyDKGResponse, error) {
+	var resp *drand.EmptyDKGResponse
 	c, err := g.conn(ctx, p)
 	if err != nil {
 		return nil, err
 	}
-	client := drand.NewDKGClient(c)
+	client := drand.NewDKGControlClient(c)
 	ctx, cancel := g.getTimeoutContext(ctx)
 	defer cancel()
-	resp, err = client.Abort(ctx, in)
+	resp, err = client.Packet(ctx, in)
 	return resp, err
 }
 
-func (g *grpcClient) Execute(ctx context.Context, p Peer, in *drand.StartExecution, _ ...grpc.CallOption) (*drand.EmptyResponse, error) {
-	var resp *drand.EmptyResponse
+func (g *grpcClient) DKGStatus(
+	ctx context.Context,
+	p Peer,
+	in *drand.DKGStatusRequest,
+	_ ...grpc.CallOption,
```

Comment:
This one isn't even part of the DKGClient interface, so you could just drop it, no?
```suggestion
```